pkg/crc/preflight/preflight_checks_tray_darwin.go

```diff
@@ -99,3 +103,12 @@ func fixPlistFileExists(agentConfig launchd.AgentConfig) error {
 	}
 	return nil
 }
+
+func checkIfDaemonRunning() error {
+	return crcerrors.Retry(context.Background(), time.Second*20, func() error {
```

Comment:
20 seconds seem very long?

Reply:
yes, it needs to be changed, in the on macOS tray we only wait 1 second before making `/status` calls, so 1 second interval and the limit we can set to 3? worst case 3 seconds wait time